index.js

```diff
@@ -1,109 +1,64 @@
 'use strict';
 
-var Plugin           = require('./lib/plugin');
-var TemplatePlugin   = require('./lib/template-plugin');
-var JavascriptPlugin = require('./lib/javascript-plugin');
-var debug            = require('debug')('ember-cli:registry');
-
-function Registry(plugins, app) {
-  this.registry = {
-    js: [],
-    css: [],
-    'minify-css': [],
-    template: []
-  };
-
-  this.instantiatedPlugins = [];
-  this.availablePlugins = plugins;
```

Comment:
@rwjblue removing `availablePlugins` unfortunately made this addon incompatible with current Ember CLI releases :-/